src/main/webapp/script.js

```diff
@@ -91,6 +91,18 @@ function loadOptions() {
       optionList.appendChild(singleOption);
     })
   });
+
+/** Creates an element that represents a comment. */
```

Comment:
```suggestion
/** Creates an element do display a comment. */
```

Reply:
Fixed! Here, and with the results display as well.